pcdsdevices/attenuator.py

```diff
@@ -1387,6 +1387,30 @@ class AT2K2(AttenuatorSXR_Ladder):
     """
 
 
+class AT3K2(AttenuatorSXR_LadderTwoBladeLBD):
+    """
+    AT1K2 solid attenuator variant from the LCLS-II L2SI project.
+
+    This has 2 blades, each with up to 8 filters each.
+    This class includes a calculator to aid in determining which filters to
+    insert for a given attenuation at a specific energy.
+
+    Parameters
+    ----------
+    prefix : str
+        Solid Attenuator base PV.
+
+    name : str
+        Alias for the Solid Attenuator.
+
+    calculator_prefix : str
+        The prefix for the calculator PVs.
+    """
+    lightpath_cpts = [f'blade_{idx:02}.state.state' for idx in range(1, 2)]
```

Comment:
range(1,2) -> [1]

If we only want to consider one blade that's fine, but we have two here?

Reply:
Setting this will also fix the status output problem here, which is currently only showing 1 blade:

```python
In [6]: a =AT3K2("AT3K2:L2SI", calculator_prefix="AT3K2:L2SI:CALC", name="sss", input_branches=["a"], output_branches=["b"])

In [9]: a
Out[9]:

+--------+----------+
| State  | blade_01 |
+--------+----------+
| Moving |          |
|  Out   |    X     |
| In_01  |          |
| In_02  |          |
| In_03  |          |
| In_04  |          |
| In_05  |          |
| In_06  |          |
| In_07  |          |
| In_08  |          |
+--------+----------+
Transmission (E=None keV): None
Transmission for 3rd harmonic (E=None keV): None

```